README.md

```diff
@@ -1,6 +1,6 @@
 # docker-postgres-plv8
 
-Docker image for running [PLV8](https://github.com/plv8/plv8) on Postgres 10.5.
+Docker image for running [PLV8](https://github.com/plv8/plv8) on Postgres 14.7.
```

Comment:
Do we need to have the Postgres version here in the first place? Seems like something that will just get out of date.